frontend/src/views/reports/AuditReporting.vue

```diff
@@ -150,28 +167,42 @@ export default {
         this.searching = false
       }
     },
-    showError(error) {
-      this.alertStore.setErrorAlert(error)
+    showError(errors) {
+      this.alertStore.setErrorAlerts(errors)
       this.result = {}
-      this.searching = false
+      this.submitting = false
     },
     resetForm() {
       this.userId = ''
       this.organizations = []
       this.transactionTypes = []
-      this.endDate = new Date()
-      this.startDate = dayjs().subtract(1, 'month').toDate()
+      this.endDate = dayjs(dayjs().startOf('month', 0).toDate()).subtract(1, 'month').toDate()
+      this.startDate = dayjs(dayjs().startOf('month').toDate()).subtract(1, 'month').toDate()
       this.result = null
       this.searchOk = false
       this.searching = false
       this.v$.$reset()
       this.alertStore.dismissAlert()
     },
+    /**
+     * Validates that End Date is after Start Date
+     */
+    validateDate() {
+      return dayjs(this.endDate).isAfter(this.startDate)
+    },
+
+    /**
+     * Validates that End Date is not more than 3 months from Start Date
+     */
+    validateDateRange() {
+      var diff = dayjs(this.endDate).diff(dayjs(this.startDate), 'month')
```

Comment:
Looks like this allows entering a range of more than 3 months e.g. 3 months and a day